client/settings/deposits/index.js

```diff
@@ -59,6 +61,22 @@ const CustomizeDepositSchedule = () => {
 		depositScheduleMonthlyAnchor,
 		setDepositScheduleMonthlyAnchor,
 	] = useDepositScheduleMonthlyAnchor();
+	const initialRender = useRef( true );
+	useEffect( () => {
+		if ( initialRender.current ) {
+			initialRender.current = false;
+			return;
+		}
+		wcpayTracks.recordEvent( 'wcpay_deposits_schedule_change', {
+			interval: depositScheduleInterval,
+			weekly_anchor: depositScheduleWeeklyAnchor,
+			monthly_anchor: depositScheduleMonthlyAnchor,
+		} );
+	}, [
+		depositScheduleInterval,
+		depositScheduleMonthlyAnchor,
+		depositScheduleWeeklyAnchor,
+	] );
```

Comment:
This event looks like a `wcpay_settings_deposits_view` event – it fires when the settings UI is rendered.

Assuming that's the case … do we need a custom event? i.e. maybe there's a wcadmin event we can use. I see `wcadmin_settings_view` and `wcadmin_page_view` firing when loading this page.

If we do need a custom `view` event here:

- Name needs to accurately reflect action, e.g. `wcadmin_settings_payments_deposits_view`.
- I don't think it should have props for the current settings – it hasn't changed yet.
  - We could add an event for logging changes to schedule, e.g. `wcpay_settings_deposit_schedule_change`. This might need to be fired from server where we can detect the settings have changed.
- The `useInitialRenderEffect` hook pattern could be a custom hook, or even better, add a named `useRenderTracksEvent` hook so we can consistently use it.
- View events are challenging here as the UI is off-screen at the start, so me might need a smarter event (that understands scroll position). 

TLDR let's use `wcadmin_settings_view` for now :)

Reply:
Noting the settings change event: `wcpay_deposit_schedule_change`